src/plugins/intel_cpu/src/emitters/snippets/x64/jit_brgemm_copy_b_emitter.cpp

```diff
@@ -19,10 +19,75 @@ using namespace Xbyak;
 using namespace dnnl::impl;
 using namespace dnnl::impl::cpu::x64;
 using namespace ov::intel_cpu::brgemm_utils;
+using namespace ov::snippets::utils;
 
 namespace ov {
 namespace intel_cpu {
 
+size_t jit_brgemm_copy_b_emitter::get_repacking_buffer_size(const ov::snippets::lowered::ExpressionPtr& copy_b_expr) {
+    const auto& in_desc = copy_b_expr->get_input_port_descriptor(0);
+    const auto& in_layout = in_desc->get_layout();
+    const auto& in_subtensor = in_desc->get_subtensor();
+    const auto& in_shape = get_planar_vdims(in_desc->get_shape(), in_layout);
+
+    const size_t n_blk = is_full_dim_value(*in_subtensor.rbegin()) ? *in_shape.rbegin() : *in_subtensor.rbegin();
+    const size_t k_blk = is_full_dim_value(*++in_subtensor.rbegin()) ? *++in_shape.rbegin() : *++in_subtensor.rbegin();
+    OPENVINO_ASSERT(!is_dynamic_value(n_blk) && !is_dynamic_value(k_blk), "get_repacking_buffer_size must be called with static subtensor values");
```

Comment:
We offline discussed helper for subtensors with full dimensions - there was idea about helper in `PortDescriptor` but we don't know about - it's input or output. What's about to add helper `get_projected_subtensor(ExpressionPort)` to `utils` file?
Then we can call this helpers:
- here
- in ctor
- https://github.com/v-Golubev/openvino/blob/a3ed68a590bf9a6fd69da7a8b9cb5c85d72c4ecd/src/common/snippets/src/lowered/port_descriptor.cpp#L142
- https://github.com/v-Golubev/openvino/blob/a3ed68a590bf9a6fd69da7a8b9cb5c85d72c4ecd/src/common/snippets/src/lowered/pass/compute_buffer_allocation_size.cpp#L63

What do you think?

Reply:
I think it's good idea, I'll try it. However, this helper can't be reused in port descriptor code because
1. We **don't touch** full dim there
2. This code is performed on ngraph node so `ExpressionPort` is inaccessible from there